conanfile.py

```diff
@@ -0,0 +1,153 @@
+#===============================================================================
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+#
+# SPDX-License-Identifier: Apache-2.0
+#===============================================================================
+
+from conans import ConanFile, CMake, tools
+from packaging.version import parse
+from six import StringIO
+
+class oneMKLConan(ConanFile):
+    name = "oneMKL"
+    version = "0.1.0-beta"
+    url = "https://github.com/oneapi-src/oneMKL"
+    description = "oneMKL interfaces is an open-source implementation of oneMKL Data Parallel C++ (DPC++) interfaces according to oneMKL specification that can work with multiple devices (backends) using device specific libraries underneath."
+    license = "Apache License Version 2.0"
+    legal_notice = "\
+LEGAL NOTICE: By downloading and using this container or script as applicable \n \
+(the “Software Package”), and any included software or software made available \n \
+for download, you agree to the terms and conditions of the software license \n \
+agreements for the Software Package, which may also include notices, \n \
+disclaimers or license terms for third party software (the “Agreements”) \n \
+located at https://github.com/conan-io/conan/blob/develop/LICENSE.md, the \n \
+THIRD-PARTY-PROGRAMS file and in the README.md file included with the Software Package. \n \
+    "
+
+    # Dependencies
+    oneapi_version = "2021.1-beta05"
```

Comment:
Shall we use beta06?

Reply:
Let's have another PR for that change. Thank you.